packages/desktop/views/dashboard/send-flow/views/components/StorageDepositButton.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+    import { Text, Icon, Modal, Toggle, FontWeight } from 'shared/components'
+    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { fade } from 'svelte/transition'
+    import { formatTokenAmountPrecise } from '@core/wallet'
+    import { getBaseToken } from '@core/profile'
+    import { localize } from '@core/i18n'
+
+    export let storageDeposit: number
+    export let giftStorageDeposit: boolean
+    export let disabled: boolean = false
```

Comment:
This is dead code, I think it's okay to remove this and only add it when we need it.

Reply:
Actually, this is not true. We need to disable it when the user has clicked confirm the transaction.